common/recipes-core/initscripts/files/hostname.sh

```diff
@@ -17,7 +17,7 @@ set_hostname() {
   # Set hostname
   hostname $name
   # Reload/Restart any services needing the updated hostname
-  /etc/init.d/syslog reload
+  /etc/init.d/syslog restart
```

Comment:
I believe we use rsyslog not syslog which does support reload:
https://github.com/facebook/openbmc/blob/helium/common/recipes-extended/rsyslog/files/initscript#L91

Reply:
True, I believe this might have been caused because of my runtime testing on qemuarmv5 (same TUNE) instead of real hardware

Reply:
I just checked and yes, the qemuarmv5 warrior build for fb-openbmc-image does not pull rsyslog (haven't checked why yet)